phoenix-core/src/main/java/org/apache/phoenix/util/SchemaUtil.java

```diff
@@ -983,14 +983,21 @@ public static boolean hasRowTimestampColumn(PTable table) {
         return SchemaUtil.getTableKey(null, schemaName, MetaDataClient.EMPTY_TABLE);
     }
 
-    public static PName getPhysicalHBaseTableName(PName pName, boolean isNamespaceMapped, PTableType type) {
-        return getPhysicalHBaseTableName(pName.toString(), isNamespaceMapped, type);
+    public static PName getPhysicalHBaseTableName(PName schemaName, PName tableName, boolean isNamespaceMapped) {
+        return getPhysicalHBaseTableName(
+            schemaName == null ? null : schemaName.toString(), tableName.toString(), isNamespaceMapped);
     }
 
-    public static PName getPhysicalHBaseTableName(byte[] tableName, boolean isNamespaceMapped, PTableType type) {
-        return getPhysicalHBaseTableName(Bytes.toString(tableName), isNamespaceMapped, type);
+    public static PName getPhysicalHBaseTableName(byte[] schemaName, byte[] tableName, boolean isNamespaceMapped) {
+        return getPhysicalHBaseTableName(Bytes.toString(schemaName), Bytes.toString(tableName), isNamespaceMapped);
     }
 
+    /**
```

Comment:
As per this comment, the 4 methods (getPhysicalTableName, getPhysicalName) can return an unexpected value. 
I don't believe this situation has been seen in the wild as-of-yet, however users should be aware of this behaviour.